packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -312,6 +312,12 @@ const OpenPositions = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    React.useEffect(() => {
+        if (active_positions.length !== 0) {
+            set_active_positions(active_positions);
+        }
+    }, [active_positions, active_positions_saved]);
```

Comment:
From what I see, this might create an infinite loop. It will set `active_positions` inside and trigger itself since `active_positions_saved` is a dependency.